okhttp-sse/src/main/kotlin/okhttp3/sse/EventSourceListener.kt

```diff
@@ -18,6 +18,11 @@ package okhttp3.sse
 import okhttp3.Response
 
 abstract class EventSourceListener {
+  /**
+   * Milliseconds elapsed between 2 events until connection failed. Doesn't timeout if null
+   */
+  open var idleTimeoutMillis: Long? = null
```

Comment:
For consistency with the rest of the library, could you make this non-nullable, using a value of 0 for no timeout?

Reply:
I don’t think the listener is the right place for this property.

Reply:
Do you mean with a setter? We need to be able to change the EventSource timeout

```
interface EventSource {
  ...
  fun timeout(timeout: Timeout)
}